src/index.test.js

```diff
@@ -316,3 +316,50 @@ test('appends selected tags to aria-labelledby with text label', t => {
   t.deepEqual(wrapper.find('.dropdown-trigger').prop('aria-labelledby'), 'rdts_trigger rdts-0_tag')
   t.deepEqual(wrapper.find('.dropdown-trigger').prop('aria-label'), 'hello world')
 })
+
+test('select correct focused node when using external state data container', t => {
+  class DropdownContainer extends React.Component {
```

Comment:
We don't need to go this far. The `setState` actions can be simulated using just the component (we do this in other tests)

Reply:
the outside setState? since is a external data change? you right will the change in other way